src/main/resources/META-INF/load_h2.sql

```diff
@@ -0,0 +1,17 @@
+INSERT INTO Plan("NAME", "DESCRIPTION") VALUES ('plan1', 'dec1')
```

Comment:
test data to the `src/test/resources` folder (if it is going to be used) otherwise, removed.

Reply:
removed